src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -18,6 +18,18 @@ namespace Uno.UI.SourceGenerators.DependencyObject
 	[Generator]
 	public class DependencyObjectGenerator : ISourceGenerator
 	{
+		private static readonly DiagnosticDescriptor _descriptor = new(
+#pragma warning disable RS2008 // Enable analyzer release tracking
+			id: "Uno0003",
+#pragma warning restore RS2008 // Enable analyzer release tracking
+			title: "Native view shouldn't implement 'DependencyObject'",
+			messageFormat: "'{0}' shouldn't implement 'DependencyObject'. Inherit 'FrameworkElement' instead.",
+			category: "TODO",
```

Comment:
@jeromelaban Any recommendation here?

Reply:
Maybe place this into a separate partial class file, as for the category, let's call it Usage for now.